course_discovery/apps/edly/api/v1/views.py

```diff
@@ -0,0 +1,104 @@
+"""
+Views for API v1.
+"""
+from django.utils.text import slugify
+from rest_framework import (
```

Comment:
rest framework is a third party import. by edx's import order conventions, they should come first. before native django imports